spec/helpers/application_helper/buttons/storage_scan_spec.rb

```diff
@@ -4,12 +4,9 @@
   let(:feature) { :smartstate_analysis }
   let(:props) { {:options => {:feature => feature}} }
   let(:button) { described_class.new(view_context, {}, {'record' => record}, props) }
-  let(:ems)                  { FactoryBot.create(:ems_vmware) }
-  let(:emss)                 { [ems] }
-  let(:emss_with_valid_auth) { [ems] }
+  let(:ems) { nil }
 
-  before { allow(record).to receive(:ext_management_systems).and_return(emss) }
-  before { allow(record).to receive(:ext_management_systems_with_authentication_status_ok).and_return(emss_with_valid_auth) }
+  before { allow(record).to receive(:ext_management_system).and_return(ems) }
```

Comment:
@himdel could you create the storage record in this context with the ems instead of stubbing the call?  Now that it is a simple belongs to that is a lot easier to do where before you'd have to link it up to hosts and host storages.

Reply:
sure, updated :)